src/NzbDrone.Api/Series/SeriesEditorModule.cs

```diff
@@ -20,10 +20,13 @@ public SeriesEditorModule(ISeriesService seriesService)
         private Response SaveAll()
         {
             //Read from request
-            var series = Request.Body.FromJson<List<SeriesResource>>().InjectTo<List<Core.Tv.Series>>();
+            var resources = Request.Body.FromJson<List<SeriesResource>>();
+
+#warning TODO: Shouldn't we somehow handle which properties can be set instead of injecting all of it?
```

Comment:
Yes, thats one thing I want in v3 to make the payload A LOT smaller. I'm not sure how we want to handle that between the client sending it and the changes being stored in the DB though.